css-ecommerce-page/index.html

```diff
@@ -0,0 +1,90 @@
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row header-arrivals">
+      <h1 class=""> New Arrivals </h1>
+    </div>
+    <div class="row">
+      <div class="column-fifth">
+        <div class="row flex-column margin-left-right">
+          <div class="columnn-fifth width ">
+            <img class="free-people-img " src="images/dress1.jpeg">
+          </div>
+          <h4 class="item-price-section">
+            <span class="brand font "> Free People </span>
+            <span class="sizes-info  font lgrey-font">Size 6</span>
+          </h4>
+          <h4 class="item-price-section">
+            <span class="prices font "> $35 </span>
+            <span class=" font lgrey-font"><s>$125 retail </s> </span>
+          </h4>
+        </div>
+      </div>
+      <div class="column-fifth">
+        <div class="row flex-column margin-left-right">
+          <div class="column-full ysl-bag">
+            <img class="row width " src="images/ysl-bag.jpeg">
+          </div>
+          <h4 class=" item-price-section">
+            <span class="brand font"> Yves Saint Lfontaurent </span>
+            <span class="sizes-info font  lgrey-font"> One-size</span></h4>
+          <h4 class=" item-price-section">
+            <span class="prices font "> $200 </span>
+            <span class=" font lgrey-font"><s>$1600 retail </s></span>
+          </h4>
+        </div>
+      </div>
+      <div class="column-fifth">
+        <div class="row flex-column margin-left-right ">
+          <div class="columnn-fifth width">
+            <img class="tiffany-key-img " src="images/key.jpeg">
+          </div>
+          <h4 class="item-price-section">
+            <span class="brand font"> Tiffany </span>
+            <span class="sizes-info font  lgrey-font"> One-size</span></h4>
+          <h4 class="item-price-section">
+            <span class="prices font"> $476 </span>
+            <span class="font lgrey-font"><s>$2000 retail </s> </span>
+          </h4>
+        </div>
+      </div>
+      <div class="column-fifth ">
+        <div class=" row  flex-column margin-left-right">
+          <div class="lv-bag-img "> </div>
+              <h4 class="item-price-section lv-words-section">
+                <span class="brand font"> Louis Vuitton </span>
+                <span class="sizes-info font lgrey-font">One-size </span>
+              </h4>
+              <h4 class="item-price-section  lv-words-section">
+                <span class="prices font "> $95 </span>
+                <span class="font lgrey-font "><s>$1200 retail </s> </span>
+              </h4>
+        </div>
+      </div>
+      <div class="column-fifth">
+        <div class="row flex-column margin-left-right">
+          <div class="columnn-fifth width">
```

Comment:
`width` is just setting this element to be full width, you should use `column-full` for that